src/contacts/ViewContact/ContactEmails/ContactEmails.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { MultiColumnList } from '@folio/stripes/components';
+
+const visibleColumns = ['email', 'description', 'language', 'categories', 'primary'];
+const columnMapping = {
+  email: <FormattedMessage id="ui-organizations.contactPeople.email" />,
+  description: <FormattedMessage id="ui-organizations.contactPeople.description" />,
+  language: <FormattedMessage id="ui-organizations.contactPeople.language" />,
+  categories: <FormattedMessage id="ui-organizations.contactPeople.categories" />,
+  primary: <FormattedMessage id="ui-organizations.primaryItem" />,
+};
+
+const ContactEmails = ({ emails }) => {
+  const resultsFormatter = {
```

Comment:
seems `formatter` can be moved out of component

Reply:
moved it out